src/Nethermind/Nethermind.Init/Steps/InitializeNetwork.cs

```diff
@@ -404,6 +417,10 @@ private async Task InitPeer()
         {
             _api.ProtocolsManager!.AddSupportedCapability(new Capability(Protocol.Snap, 1));
         }
+        
+        // Add etha protocol capability
+        _api.ProtocolsManager!.AddSupportedCapability(new Capability(Protocol.Etha, 1));
+        
```

Comment:
It is unconditional, so should be added to default capabilities. So remove from here, add in `ProtocolsManager.DefaultCapabilities`, adjust tests